gov2/dynamodb/README.md

```diff
@@ -54,20 +53,6 @@ predictable performance with seamless scalability.*
   
 ## Running the examples
 
-### Get started using tables, items, and queries
-
-This interactive scenario runs at a command prompt and shows you how to use DynamoDB
-to do the following:
-
-- Create a table that can hold movie data.
-- Put, get, and update a single movie in the table.
-- Write movie data to the table from a sample JSON file.
-- Query for movies that were released in a given year.
-- Scan for movies that were released in a range of years.
-- List tables in your account.
-- Delete a movie from the table.
-- Delete the table.
-
```

Comment:
Keep these breakout descriptions of individual examples in the README.

Reply:
restored